migrations/sql/R__code-table-data.sql.sql

```diff
@@ -40,6 +40,4566 @@ set
+    'User/service - Agriculture - SGARs',
+    645,
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  ),
+  (
+    'USAQUACULT',
+    'USAQUACULT',
+    'User/service - Aquaculture',
+    655,
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  ),
+  (
+  'USAFMGTION',
+  'USAFMGTION',
+  'User/service - Fumigation',
+  695,
+  'Y',
+  CURRENT_USER,
+  CURRENT_TIMESTAMP,
+  CURRENT_USER,
+  CURRENT_TIMESTAMP
+ ),
+ (
+  'USINDSTVEG',
+  'USINDSTVEG',
+  'User/service - Industrial vegetation',
+  710,
+  'Y',
+  CURRENT_USER,
+  CURRENT_TIMESTAMP,
+  CURRENT_USER,
+  CURRENT_TIMESTAMP
+ ),
+ (
+  'USSTRCSGAR',
+  'USSTRCSGAR',
+  'User/service - Structural – SGARs',
+  865,
+  'Y',
+  CURRENT_USER,
+  CURRENT_TIMESTAMP,
+  CURRENT_USER,
+  CURRENT_TIMESTAMP
+ )
+ ON CONFLICT
+ DO NOTHING;
+
+-- add schedule_sector_xrefs
+
+INSERT INTO
+  case_management.schedule_sector_xref (
+    schedule_sector_xref_guid,
+    schedule_code,
+    sector_code,
+    active_ind,
+    create_user_id,
+    create_utc_timestamp,
+    update_user_id,
+    update_utc_timestamp
+  )
+VALUES
+  (
+    uuid_generate_v4 (),
+    'IPM',
+    'USAERIALDR',
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  ),
+  (
+    uuid_generate_v4 (),
+    'IPM',
+    'USRAGRICU',
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  ),
+  (
+    uuid_generate_v4 (),
+    'IPM',
+    'USAGRISGAR',
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  ),
+  (
+    uuid_generate_v4 (),
+    'IPM',
+    'USAQUACULT',
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  ),
+  (
+    uuid_generate_v4 (),
+    'IPM',
+    'USAFMGTION',
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  ),
+  (
+    uuid_generate_v4 (),
+    'IPM',
+    'USINDSTVEG',
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  ),
+  (
+    uuid_generate_v4 (),
+    'IPM',
+    'USSTRCSGAR',
+    'Y',
+    CURRENT_USER,
+    CURRENT_TIMESTAMP,
+    CURRENT_USER,
+    CURRENT_TIMESTAMP
+  )
+   ON CONFLICT DO NOTHING;
+
+
+-- The following codes are no longer relevant.  They can be deleted rather than inactivated as 
+-- we aren't live yet.
+
+delete from case_management.schedule_sector_xref 
+where sector_code IN 
+ (
+ 'SGENRODEN',
+ 'USRAGRGEN',
+ 'USRAGGENE',
+ 'USRFORENU', 
+ 'USRFORE50H', 
+ 'USRFORE500', 
+ 'USRFOREMOR', 
+ 'USRFUMCON', 
+ 'USRFUMSHS', 
+ 'USRINDPAVE', 
+ 'USRMOSAEBA', 
+ 'USRMOSGRBA', 
+ 'USRNETTRE', 
+ 'USRPESTNS', 
+ 'USRPESTPL', 
+ 'USRPESTSE', 
+ 'USRFORMNB',
+ 'USRFORNU2', 
+ 'USRFORSEO',
+ 'USRFUMCO2',  
+ 'VENDGENERA',
+ 'VENDGENLAN'
+ );
+
+delete from  case_management.sector_code 
+where sector_code IN 
+ (
+ 'SGENRODEN',
```

Comment:
After running the script on my local env these sector-code entries are not being removed so all the ones that are capitalized in the DB are still there for me. They AREN'T visible to the end user when viewing the dropdowns though so likely not a big problem.